### PR TITLE
WIP: Fix navbar on blog pages

### PR DESCRIPTION
**Pull request by afolson**

### CODE REVIEW

**Comment by afolson** [comment]
@cwjames19 Are you able to take a look at this one? On the live site, the menu pops over blog posts (it works as intended on other pages): https://www.swimos.org/blog/2024/1/16/fine-grained-streaming-apis-from-a-kafka-topic.html

I fixed some things in the header (references to colors we aren't using) but I'm now not able to get the menu to render on blog post pages: 

Screenshot 2024-03-25 at 4 56 55 PM

I think this has something to do with Alpine.js and x-open/x-cloak but I couldn't figure out what the exact issue is. It seems like open: true is being set somehow but the menu isn't actually rendering. I can get other stuff to appear consistently when open: true is set, I just can't get the menu to work. It didn't seem to be a priority thing either. Thanks!

**Comment by cwjames19** [comment]
@afolson No problem! I'll take a look.

**Comment by cwjames19** [comment]
@afolson Figured it out. Took a while but it turned out the Alpine.js scripts just weren't being loaded on the blog pages so I moved them from `footer-scripts.html` to `head.html`. If it looks good to you, feel free to merge.

**Comment by afolson** [comment]
@cwjames19 Thanks a ton!

File: _includes/header.html [hunk 1]
```diff
@@ -10,16 +15,41 @@
       <!-- Desktop menu -->
       <div class="hidden lg:flex">
       {% for section in site.data.header-nav %}
-        <div x-cloak x-data="{ open: false }" class="group flex" @mouseenter="open = true" @mouseleave="open = false">
+        <div
+          x-cloak
+          x-data="{ open: false }"
+          class="TEST-FOO group flex"
```
Comment (afolson) [inline _includes/header.html:21]: @cwjames19 Can I delete TEST-FOO from here?

Comment (cwjames19) [reply _includes/header.html:21]: @afolson Whoops, yes of course

**Pull request merged**